### ISSUE TITLE
Specifying window's title while debugging

### ISSUE THREAD

**Issue by sowee15** [Microsoft/nodejstools#209]
Hi,

Is there a way to specify the title of an app (what is shown in the title bar and in the title column in the process explorer?) I know I can set one using start.exe but is there a way while starting instances from visual studio? Thanks

**Comment by mousetraps** [comment]
Have you tried ```process.title = "my title"```?

**Comment by sowee15** [comment]
@mousetraps, I was sure I had tried that before, maybe something was changed in regards to that in nodejstools? Now it works partially: it sets the title on the console app, it sets the title in the debug > attach to process menu but it does not set the title in the debug > windows > processes window... This last spot would be the most useful one to me since its from there that I quickly detach from one process to then stop it and restart it. I currently rely on the PID to do that.

I should probably edit my original post to specify that the title should also be available in the process window, I thought it would be the case if the title bar was set.
